### ISSUE TITLE
Instances of a service on different nodes not connecting to LBs

### ISSUE THREAD

**Issue by jackewitz** [kontena/kontena#1246]
Dear Developer & Community,

I've got a problem with the connectivity between instances of services on different nodes to LBs (kontena version 0.15.5).

There a 3 nodes and if a run 3 instances of LB and 3 instances of the service (every instance on a different node) I can see in the logs of the LBs:
- on node 1 the instance 1 of the services connects to the instance 1 of the LB
- on node 2 the instance 2 of the services connects to the instance 2 of the LB
- on node 3 the instance 3 of the services connects to the instance 3 of the LB

Hmmm? I suggested, every instance of the service will be connected to every instance of the LB. Am I wrong?

Next try: 3 nodes, 3 instance of a service, just 1 instance of LB -> LB is on node 1. Logs of the LB said:
- on node 1 the instance 1 of the services connects to the instance 1 of the LB
- No log entries for instance 2 and 3 of the service in LB logs.

Hmm? A bug in Kontena? A bug in the configuration of Kontena (or underlying tools)? A bug in the kontena.yml of the service? Or a "bug" on my own - maybe a understand something wrong?

I read the docs of load balancing in Kontena but multi-node-load-balancing is not mentioned in there. Or have I miss someting? 

Ciao .. Iver
## kontena.yml

> ---
> 
> version: '2'
> name: service
> services:
>   web:
>     image: registry.staging.kontena.local/service:latest
>     instances: 3
>     links:
>       - lb
>     environment:
>       - KONTENA_LB_INTERNAL_PORT=3000
>       - KONTENA_LB_VIRTUAL_PATH=/
>   lb:
>     image: kontena/lb:latest
>     instances: 1
>     ports:
>       - 80:80

**Comment by jakolehm** [comment]
LB should balance across nodes. How did you install Kontena?

**Comment by jackewitz** [comment]
I used VMware with Ubuntu 14.04.05.

It seems that there are 1 etcd-cluster with 1 etcd-node on every kontena-node. So etcd don't share the data / informations between the kontena-nodes.

I think the mistake was, that I cloned the initial VM (kontena-node-0) after installing docker. After cloning and install kontena (with etcd) there was an issue with unique docker-ids. After changing the docker id at kontena-node-1 and kontena-node-2, kontena-master noticed 3 nodes but etcd was installed before, so .. maybe this is the failure.

I just will remove the current installation of kontena 0.15.5 and started with a clean setup.
